src/modules/beamos/filesystem/migration/usr/bin/migration.sh

```diff
@@ -337,6 +338,13 @@ do_restore_data () {
   echo "$(timestamp) $0: Restoring plugins.mrbeam.review for $configfile"
   sudo cat $backupfile | sudo yq ea -i 'select(fileIndex==0) * {"plugins":{"mrbeam":{"review":select(fileIndex==1).plugins.mrbeam.review}}}' $applyfile -
 
+  #plugins.swupdater.attributes.migration-operator
+  echo "$(timestamp) $0: Save plugins.swupdater.server_url and plugins.swupdater.server_port from $applyfile"
+  server_url=$(sudo yq eval '.plugins.swupdater.server_url' $applyfile)
+  server_port=$(sudo yq eval '.plugins.swupdater.server_port' $applyfile)
+  echo "$(timestamp) $0: Set plugins.swupdater.attributes.migration-operator as $migrationoperator for $configfile"
+  sudo yq e -i ".plugins.swupdater |= {\"server_url\": \"$server_url\", \"server_port\": $server_port, \"attributes\": {\"migration-operator\": \"$migrationoperator\"}}" $applyfile
+
```

Comment:
1- Why do we need to add the server URL and port? These will be added by the SW updater plugin anyway, no?
2- Why are server_url and server_port with an underscore but migration-operator with a dash?

Reply:
1. We use "yq" command to modify yaml file. In this issue, we would like to add attributes section but unfortunately yq tool does not support adding a section. We can add another line like server_port but adding a new section (they call a map) is not supported by the yq.
2. We decided to use dash.